README.md

```diff
@@ -1,18 +1,89 @@
+* IRC: permission to change the topic in [#theforeman on Libera.Chat)[ircs://irc.libera.chat/theforeman). See [Libera.Chat permissions](https://libera.chat/guides/creatingchannels#setting-up-permissions) on how channel owners can grant this.
+
+#### Katello release owner
+
+* Write permission to [Katello/katello](https://github.com/Katello/katello)
+* Push access to the [Katello gem](https://rubygems.org/gems/katello)
+* [Discourse](https://community.theforeman.org/) account level 3 to post to [Release announcements](https://community.theforeman.org/c/release-announcements/8)
+
+#### Katello release engineer
+
+These are the same as Foreman's release engineer.
+
+### Branching
+
+Use `procedure_branch` to display the steps. Modify `PROJECT` and `VERSION` as needed.
+
+```sh
+PROJECT=foreman VERSION=3.7 ./procedure_branch
+```
+
+It will have created `releases/${PROJECT}/${VERSION}/settings`, which should be submitted as a pull request.
+
+A more advanced invocation is to specify the parameters directly and copy the output. This uses `wl-copy` for Wayland; `xclip` achieves the same for Xorg.
+
+```
+PROJECT=foreman VERSION=3.7 ./procedure_branch 2023-05-23 TheOwner TheEngineer | wl-copy
+```
+
+Now post this to [Discourse Development/Releases](https://community.theforeman.org/c/development/releases/20) as `$PROJECT $VERSION branching process`. Follow the process as instructed.
+
+### Release
+
+First ensure `releases/${PROJECT}/${VERSION}/settings` contains the correct `FULLVERSION`. Modify as needed and submit as a pull request. Then generate the procedure, similar to the branching process:
+
+```sh
+PROJECT=foreman VERSION=3.7 ./procedure_release
+```
+
+Or the complete version:
+```
+PROJECT=foreman VERSION=3.7 ./procedure_release 2023-05-23 TheOwner TheEngineer | wl-copy
+```
+
+Now post this to [Discourse Development/Releases](https://community.theforeman.org/c/development/releases/20) as `$PROJECT $FULLVERSION release process`. Follow the process as instructed.
 
 ## Setup
 
-Required dependencies:
+A release engineer needs:
 
  * gopass
  * python3
  * python-jenkins
 
+On Fedora:
+
+```sh
+dnf install gopass python3-jenkins
+```
+
 ### Gopass
 
-First, clone this repository to your local machine. [gopass](https://github.com/gopasspw/gopass) will need to be installed and then set up on your machine for the release environment.
+First, clone this repository to your local machine. [gopass](https://github.com/gopasspw/gopass) will need to be installed and then set up on your machine for the release environment. You will also need [access](https://theforeman.github.io/foreman-infra/#access) to our infrastructure.
```

Comment:
somewhere should be written that $yourkey needs to be in the list of gopass recipients, which it won't be for new users who just followed `#access`

Reply:
I think I've addressed this now in the Roles -> Foreman release engineer part. Can you confirm?